test/webpack.config.js

```diff
@@ -9,6 +10,7 @@ const webpackConfig = {
     filename: `tests.bundle.js`,
     pathinfo: true,
   },
+  devtool: 'eval-source-map',
```

Comment:
You might want to make this `inline-cheap-module-source-map`, it should be slightly faster with the only downside (afaik) being error traces will not link to exact character in line (column) but will still link to the correct line.

`inline` - not sure what the test runner is here, but apparently this is required by mochapack (runner we use for NodeJS tests in analytics) in order for the debugging experience to work properly
https://github.com/sysgears/mochapack/blob/master/docs/installation/webpack-configuration.md
![Screen Shot 2020-05-15 at 9 48 31 AM](https://user-images.githubusercontent.com/299587/82075737-91471300-9691-11ea-8feb-a46f8e96b126.png)